hibernate-core/src/main/java/org/hibernate/dialect/identity/CockroachDB1920IdentityColumnSupport.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.dialect.identity;
+
+import java.sql.Types;
+
+public class CockroachDB1920IdentityColumnSupport extends IdentityColumnSupportImpl {
+    @Override
+    public boolean supportsIdentityColumns() {
+        // Full support requires setting the sql.defaults.serial_normalization=sql_sequence in CockroachDB.
+        return false;
+    }
+
+    @Override
+    public String getIdentitySelectString(String table, String column, int type) {
+        return "select currval('" + table + '_' + column + "_seq')";
```

Comment:
Do we need some escaping mechanism to guard against the possibility that 'table', 'column' names might contain special characters (like "(", ")", "'", etc.)?

Reply:
This implementation is the same as PostgreSQL81IdentityColumnSupport

However, since the CockroachDB dialect does not support identity columns, it would be more clear if I just removed these method overrides.